src/Protocol/ActivityPub/Receiver.php

```diff
@@ -1157,11 +1159,12 @@ public static function getReceiverURL(array $activity): array
 	 * @param array  $tags
 	 * @param bool   $fetch_unlisted
 	 * @param bool   $push
+	 * @param string $http_signer
 	 *
 	 * @return array with receivers (user id)
 	 * @throws \Exception
 	 */
-	private static function getReceivers(array $activity, string $actor, array $tags, bool $fetch_unlisted, bool $push): array
+	private static function getReceivers(array $activity, string $actor, array $tags, bool $fetch_unlisted, bool $push, string $http_signer): array
```

Comment:
It is a private function and I added the parameter at each of the 3 calls.

Reply:
It's a code philosophy request: since the parameter is checked for emptiness in the method's body, it shouldn't be mandatory in the parameter list to manage expectation. It also matches the `getBestUserForActivity()` prototype where the `$http_signer` is passed as is to `getReceivers()`, so it makes sense they would be declared the same.